src/Shouldly.Tests/InternalTests/DifferenceIndexConsolidatorTests.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using System.Collections.Generic;
-using NUnit.Framework;
 using Shouldly.DifferenceHighlighting;
+using Xunit.Extensions;
 
 namespace Shouldly.Tests.InternalTests
 {
-    [TestFixture]
     public class DifferenceIndexConsolidatorTests
     {
         const int MaxDiffLength = 5;
         const int MaxLengthOfStrings = 21;
 
-        [Test, TestCaseSource("IndexConsolidationTestCases")]
+        [Theory]
+        [PropertyData("IndexConsolidationTestCases")]
```

Comment:
This is something I am not done with, I am using the RC of the new NuGet package manager and I can't install xunit 2.0... I also wrote most of this on a plane so I manually shoehorned it in. I will fix properly and use xunit 2.0 before I complete this PR